palette/tests/hsluv_dataset/hsluv_dataset.rs

```diff
@@ -0,0 +1,65 @@
+//! rev4 of the verification dataset from HSLuv
+//! https://github.com/hsluv/hsluv/blob/master/snapshots/snapshot-rev4.json
+
+use lazy_static::lazy_static;
+use approx::assert_relative_eq;
+use serde_json;
+
+use palette::convert::IntoColorUnclamped;
+use palette::white_point::D65;
+use palette::{Lch, Luv, Xyz};
+use std::collections::HashMap;
+
+#[derive(Clone, Debug)]
+struct HsluvExample {
+    name: String,
+    lch: Lch<D65, f64>,
```

Comment:
Is this the correct LCH? 😬

Reply:
I believe this is the appropriate LCH, but I'll remove it until I actually implement LCH in the next round so it's actually tested.

Reply:
Silly me. That's *definitely* not the right `Lch`. I started working on `Lchuv` in a new branch on top of this one, and mentally I've already changed this. But of course it's not right in this branch yet 🤦🏾

Reply:
Parallel universes can be confusing like that. 😁